engine/src/org/pentaho/di/trans/steps/sort/SortRowsMeta.java

```diff
@@ -320,7 +341,8 @@ public void getFields( RowMetaInterface inputRowMeta, String name, RowMetaInterf
         ValueMetaInterface valueMeta = inputRowMeta.getValueMeta( idx );
         valueMeta.setSortedDescending( !ascending[i] );
         valueMeta.setCaseInsensitive( !caseSensitive[i] );
-
+        valueMeta.setCollatorDisabled( !collatorEnabled[i] );
```

Comment:
This kind of construct is awkward. You should stick with either collatorEnabled or collatorDisabled across the board. It's confusing when debugging to see xxxDisabled in one place, and then xxxEnabled in another.